scipy-1.0/paper.tex

```diff
@@ -122,6 +122,21 @@ \section*{Project organisation and community}
 
 \textbf{Governance}
 
+SciPy adopted an official Governance Document on August 3, 2017 (see PR\#6953). In summary,
+we have a Benevolant Dictator for Life (BDFL) in Pauli Virtanen, and a (currently) 18
+member Steering Council chaired by Ralf Gommers that i.e., decides on who gets commit rights.
+
+SciPy adopted an official Code of Conduct on October 24, 2017 (see PR\#7963), drawing
```

Comment:
@online{SciPyCodeOfConduct,
  author = {The SciPy Community},
  title = {{SciPy} Code of Conduct},
  year = 2017,
  url = {https://docs.scipy.org/doc/scipy/reference/dev/conduct/code_of_conduct.html},
  urldate = {2018-06-01}
}

and then of course we can \cite{SciPyCodeOfConduct} in the text.